Form/Type/FileType.php

```diff
@@ -56,7 +56,8 @@ public function __construct($class, UploadFileHelperInterface $uploadFileHelper)
      */
     public function getParent()
     {
-        return 'file';
+        $type = method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix') ? 'Symfony\Component\Form\Extension\Core\Type\FileType' : 'file';
```

Comment:
I wonder if there is a smoother way to determine the SF version, /cc @WouterJ @dbu